examples/imdg-connectors/src/main/java/com/hazelcast/jet/examples/imdg/MapSourceAndSinks.java

```diff
@@ -65,17 +65,20 @@ private static Pipeline mapSourceAndSink(String sourceMapName, String sinkMapNam
     }
 
     /**
-     * This will take the contents of source map, converts values to the string and
-     * suffixes the value with {@code odd} if the key is odd and with {@code event} if the key is even.
+     * This will take the contents of source map and suffixes the value with {@code odd} if the key is odd and with
```

Comment:
- We wrap javadoc to 72 characters net. 
- We can also say "This will take the contents of **a** map", i.e. not use "source", because there's no source and sink map, just one map.